.changeset/itchy-cherries-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-hive/gateway-runtime': minor
+---
+
+New plugin: Demand Control
```

Comment:
I know these notes are for internal, but I didnt know what "demand control" was for. It wasnt intuitive that it's to figure out the cost of requests.

Reply:
I'll update there when docs are ready!